hub/core/chunk_engine/read.py

```diff
@@ -1,6 +1,6 @@
 import os
-import pickle
 import numpy as np
+import pickle  # TODO: NEVER USE PICKLE
```

Comment:
What a great way to set the tone for a review :joy:

Reply:
lol the TODO for pickle just keeps getting moved and elevated in aggression